src/assets/js/components/User.js

```diff
@@ -83,11 +97,36 @@ class User extends React.Component {
   }
 
   render() {
-    const {handleSubmit, pristine, reset, submitting} = this.props;
+    const {handleSubmit, pristine, reset, submitting, onDeleteUser, user} = this.props;
 
     return (
       <div>
-        <h3>User <small>{this.props.user._id}</small></h3>
+        <div className="row">
+          <div className="col-md-6">
+            <h3>User <small>{this.props.user._id}</small></h3>
+          </div>
+          <div className="col-md-6">
+            <div className="float-right">
+              {this.state.showConfirm ?
```

Comment:
Might want to move all of this into its own method, and then call it conditionally. Could be re-used elsewhere.

Reply:
What exactly are you talking about? The logic to show the confirmation buttons?

Reply:
Yeah the render logic, at least. Just to keep things clean